gtfs/spec/en/reference.md

```diff
@@ -295,6 +310,70 @@ For examples that demonstrate how to specify a fare structure with [fare_rules.t
+| `spanning_limit` | Non-negative integer | **Conditionally Forbidden** | Defines how many consecutive legs the transfer rule may be applied to.<br><br>Valid options are:<br>`0` or empty - No limit.<br>`2` or more - Defines how many legs the transfer rule may span.<br><br>Conditionally Forbidden:<br>- **Forbidden** if `fare_transfer_rules.from_leg_group_id` does not equal `fare_transfer_rules.to_leg_group_id`.<br>- Optional if `fare_transfer_rules.from_leg_group_id` equals `fare_transfer_rules.to_leg_group_id`. |
+| `duration_limit` | Non-negative integer | Optional | Defines the duration limit of the transfer.<br><br>Must be expressed in integer increments of seconds.<br><br>If there is no duration limit, `fare_transfer_rules.duration_limit` must be empty. |
+| `duration_limit_type` | Enum | **Conditionally Required** | Defines the relative start and end of `fare_transfer_rules.duration_limit`.<br><br>Valid options are:<br>`0` - Between the departure fare validation of the first leg and the arrival fare validation of the last leg.<br>`1` - Between the departure fare validation of the first leg and the departure fare validation of the last leg.<br>`2` - Between the arrival fare validation of the first leg and the departure fare validation of the last leg.<br>`3` - Between the arrival fare validation of the first leg and the arrival fare validation of the last leg.<br><br>Conditionally Required:<br>- **Required** if `fare_transfer_rules.duration_limit` is defined.<br>- **Forbidden** if `fare_transfer_rules.duration_limit` is empty. |
+| `fare_transfer_type` | Enum | **Conditionally Required** | Indicates the cost processing method of a sub-journey.<br><br>Valid options are:<br>`0` - The cost of the sub-journey is the total calculated cost of the preceding leg(s) and transfer(s) plus the cost of `fare_transfer_rules.amount`.<br>`1` - The cost of the sub-journey is the total calculated cost of the preceding leg(s) and transfer(s) plus the cost of `fare_transfer_rules.amount` plus the cost of `fare_leg_rules.amount` for `fare_transfer_rules.to_leg_group_id`.<br>`2` - The cost of the sub-journey is the cost of the leg with the most expensive `fare_leg_rules.amount` plus the cost in `fare_transfer_rules.amount`.<br>`3` - The cost of the sub-journey from the preceding leg(s) and transfer(s) to the transferred leg is the cost in `fare_transfer_rules.amount`.<br><br>Conditionally Required:<br>- **Required** if `fare_transfer_rules.amount` is defined.<br>- **Forbidden** otherwise. |
```

Comment:
> `1` - The cost of the sub-journey is the total calculated cost of the preceding leg(s) and transfer(s) plus the cost of `fare_transfer_rules.amount` plus the cost of `fare_leg_rules.amount` for `fare_transfer_rules.to_leg_group_id`.

Since `fare_transfer_rules.to_leg_group_id` is optional, and not required to use `fare_transfer_type=1`, this should be rephrased to something like:
>`1` - The cost of the sub-journey is the total calculated cost of the preceding leg(s) and transfer(s) plus the cost of `fare_transfer_rules.amount` plus the cost of `fare_leg_rules.amount` for the post-transfer leg.

Reply:
+1